.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: php
 
+sulu: false
```

Comment:
It should force container builds, which are faster: http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure